docs-site/src/markdown-pages/design/technical-design.md

```diff
@@ -0,0 +1,175 @@
+for bring the data captured up to new standards for meta-data that are not
+caught in existing systems.
+
+This type of component is a long term vision for the EOP solution.
+
+Example Capture components:
+
+- Field capture user interface that may be running on a mobile device
+- IOT endpoints for capturing data from sensors
+- User interface for QA review
+
+## Data Stores
+
+Data store components are about storing data in a way that is appropriate for
+the type of analysis being performed. The data storage type being used drives
+the types of queries that can be performed and the type of information that can
+be derived. By using the hub and spoke model with Kafka as the hub, EOP can stay
+agnostic of the data stores that are used, rahter than tying the solution to a
+specific type of analysis.
+
+Example Data store components:
+
+- OLTP database for fast operational access
+- OLAP database for slower analytical access
+- Graph database for network analysis
+
+## Processing
+
+Processing components are there to perform analysis on the data that is stored
+in EOP.
+
+The processing components are loosely coupled to the data stores. The intention
+is for this to be a key extension point for adding new analysis processes to EOP
+without changing other components.
+
+Example Processing components:
+
+- Real time processing has driven off kafka streams
+- Batch processing driven data in the OLAP data stores
+- Data science scripts (R / Python) for analysis e.g. Naturalised flow
+- AI model training
+
+## Output
+
+Output components systems take information captured in EOP and present it to
+users as usable outcomes. The diagram shows the output components grouped by the
+type of user that they are intended for. That is to draw attention to the fact
+that delivering an outcome may be presented differently depending on the
+consumers even though they may be in the same type of technology.
+
+Example Processing components:
+
+- UI Viewers
+- API endpoints
+- Bulk data downloads
+
+### External Applications B2C
+
+Applications that general public are intended to consume, for these there needs
+to be a focus on the user experience making the tools valuable for the user.
+
+### External Applications B2B
+
+Applications that are intended to be used by third party organisations including
+central government. The information returned may have a focus on providing for
+specific questions. These types of application may be more focused on
+information exported for use in third party systems
+
+### Internal Tools
+
+Tools that are used by council staff to perform their day to day work. These
+applications need a user experience that is task focused for the type of person
+using them and may expect a higher level of expertise when understanding the
+data.
+
+### Feeds to Council Systems
+
+Feeds to council systems where EOP has created information that is then fed back
+to existing council systems for presentation
```

Comment:
```suggestion
Output components present information captured and generated by EOP to various
types of users, with a focus on outcomes.

The diagram shows these grouped by the type of user emphasise a user-centric,
rather than technology-centric approach. E.g. multiple web applications or APIs
may be developed to serve difference audiences.

 Example Processing components:

- UI Viewers
- API endpoints
- Bulk data downloads

### External Applications B2C

Applications intended for public use. These should be focussed on presenting 
information and outcomes in a user-friendly way.

### External Applications B2B

Applications intended for use by third-party organisations including central
government. This could be information for particular types of reporting, often
provided as exports for use in third party systems.

### Internal Tools

Tools used by council staff to perform day-to-day work. These might task-focussed
applications, developed for users with particular expertise in the data.

### Feeds to Council Systems

These may be useful where EOP has created information that is valuable for
presentation by existing council systems.
```